mig/unittest/testcore.py

```diff
@@ -28,17 +28,57 @@
 """Unit tests for core helper functions"""
 from __future__ import print_function
 
+from configparser import ConfigParser
```

Comment:
Hmm, I think I recall this not being directly python2 compatible ... ConfigParser vs configparser. Leaving for now as tests seem to succeed, but it might cause trouble at some point.

Reply:
No, it seems to be one of those backports or additions from `future` just not available in the stripped-down python2 on ubuntu22.04 where I checked.